### ISSUE TITLE
Seems to break regular eslint CLI

### ISSUE THREAD

**Issue by rohan-deshpande** [IanVS/eslint-nibble#40]
Hey awesome package, really useful for going through a big codebase rule by rule and fixing things. 

However, I noticed that that after installing it, regular `eslint` linting via the CLI is not observing the `.eslintrc` properly. 

Here's an example; prior to installing `eslint-nibble` the following code, correctly as per my config, does not throw an eslint error

```
static propTypes = {
  actions: PropTypes.object,
  children: PropTypes.any,
}
```

However, after installing `eslint-nibble` and then simply running `eslint` on this file I see 

```
18:10 error 'propTypes' is not defined no-undef
```

Which is incorrect as per my rules. Running `eslint-nibble` however, correctly does not deem this to be an error. What's going on here?

Uninstalling `eslint-nibble` restores the ability for the regular `eslint` CLI command to lint correctly.

**Comment by IanVS** [comment]
Interesting.  The only thing I can think of, is that when you install `eslint-nibble`, it brings in a new version of `eslint`, since it includes `eslint` in its dependencies.  

To test, could you try something out?  Before installing `eslint-nibble`, find out what version of eslint is installed with `yarn list eslint` or `npm ls eslint`, then install `eslint-nibble` and check again.  If the version changes, try to remove `eslint-nibble` again, and then update your `eslint` to the latest version, and see if the error returns.

If that's barking up the wrong tree, could you share a bit more of the code?  It's unclear to me where the error is actually occurring.  That error shouldn't be thrown for a definition `static propTypes =`, so my _guess_ is that maybe you're referencing it somewhere by `proptypes` instead of `this.propTypes`.

Anyways, I'm glad the tool is useful for you, and please let me know what you find out after checking versions.

**Comment by ljharb** [comment]
@IanVS `eslint` should not be a dep of eslint-nibble, it should be a `peerDep` - that way you'll force it to be installed, but you'll use the version that is already installed.

**Comment by IanVS** [comment]
Thanks @ljharb. In my experience peer dependencies can be really confusing for new users to deal with, especially since npm and yarn don't install them automatically, and don't always do a great job of alerting the user what's needed.  My goal for this package was to make things as simple as possible to get started with eslint, and once up-and-running, for this package to be removed and replaced with a standard eslint setup.  And if eslint was already installed I assumed it would be deduped correctly and having it as a dependency of nibble wouldn't be harmful.

But perhaps it's time I rethink that approach.  A peerDep on eslint is probably the right thing, I just wish the ergonomics were better.

**Comment by ljharb** [comment]
Alternatively, you could use a dep *and* a peer dep; which would only complain when there was a conflict.
